src/anemoi/training/commands/profiler.py

```diff
@@ -29,19 +31,55 @@ class Profile(Command):
     def add_arguments(parser: argparse.ArgumentParser) -> argparse.ArgumentParser:
         return parser
 
-    @staticmethod
-    def run(args: list[str], unknown_args: list[str] | None = None) -> None:
-        del args
+    def run(self, args: list[str], unknown_args: list[str] | None = None) -> None:
+        # This will be picked up by the logger
+        os.environ["ANEMOI_PROFILER_CMD"] = f"{sys.argv[0]} {args.command}"
```

Comment:
Does this need to be a specific one for the profiler? I think we can just reuse the ANEMOI_TRAINING_CMD env var?

Reply:
Yes I wanted to check that! I first opted to have the two of them just to check if it's was working fine, which it does! Right now there is also quite a bit of repeated code across the profiler and train command. So I was thinking I could directly inherit from Train to do the Profiler one to avoid repeating the _merge_sysargv and other functions? the command could even go to a small function so then if I inherit I don't need to code it again. What do you think? (I have not looked  a lot to the details of the Command class, so would like to check thoughts in inheritance could be okey or is not advised in this case)